tools/morpheus/macros.xml

```diff
@@ -0,0 +1,77 @@
+<macros>
+        <token name="@TOOL_VERSION@">2.255.0</token>
```

Comment:
the version needs to be bumped

Reply:
agreed! The latest version is 288. do you want me to bump it up to 288?

Reply:
Thats a good idea.

Reply:
@bgruening I bumped the version up to 288 from 255 and updated the output of the test section. Can you check again.